{{cookiecutter.repostory_name}}/envs/prod/.vuln.env.template

```diff
@@ -0,0 +1,5 @@
+ENV=prod
+DD_URL=
```

Comment:
let's add the default url here

Reply:
I don't think exposing urls of our infrastructure services in the open source repository is a good idea

Reply:
okay maybe let's leave a hint where to look

Reply:
Added a link to the internal handbook in the README_AWS